frontend/packages/console-app/src/components/guided-tours/TourItemDescription.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import './TourItemDescription.scss';
+
+type TourItemDescriptionProps = {
+  description: string;
+  prerequisites: string[];
+};
+const TourItemDescription: React.FC<TourItemDescriptionProps> = ({
+  description,
+  prerequisites,
+}) => (
+  <>
+    <div className="odc-tour-item-description--section">{description}</div>
+    <div className="odc-tour-item-description--section">
+      <span className="odc-tour-item-description--boldtext">Prerequisites</span>
```

Comment:
Can this be replace with a header? Not sure which PF aligns (if any).
Maybe `<h6>` ?

Reply:
updated to <h5>